users.html

```diff
@@ -878,6 +938,9 @@ <h1 id="page-title" ng-non-bindable="">GEOS-Chem Users</h1>
   // Imperial College London (Sebastian Eastham)
   L.marker([51.49882885072645, -0.17491641783162853], { icon: greenIcon }).addTo(mymap).bindPopup("<strong>Imperial College London</strong><br>London, UK<ul><li><a href='https://imperial.ac.uk/people/s.eastham' target='_new'>Brahmal Vasudevan Institute for Sustainable Aviation</a></li></ul>");
 
+  // James Cook University (Jenny Fisher)
+  L.marker([-19.32586860621464, 146.75661738108985], { icon: greenIcon }).addTo(mymap).bindPopup("<strong>University of Wollongong</strong><br>Wollongong, New South Wales, Australia<ul><li><a href='https://portfolio.jcu.edu.au/Researchers/jenny.fisher#cc_researcher-research' target='_new'>Fisher Group</a></li></ul>");
```

Comment:
Just one thing, the comment still says Wollongong.  Should say "Douglas, Queensland, Australia"

Reply:
Now fixed in https://github.com/geoschem/geoschem.github.io/pull/44/commits/32d4f6009d52815ed2eedf1f689fbd0627674197